platform/authorized_api/apps/index.json

```diff
@@ -329,8 +356,8 @@
           "example": true
         },
         "gdprUserModeEnabled": {
-          "description": "Gdpr User Mode enabled or disabled for an app",
           "type": "boolean",
+          "description": "Gdpr User Mode enabled or disabled for an app",
```

Comment:
Looks like moving descriptions in these params was not introduced in backport fixes.

Reply:
both descriptions are for params (`gdprUserModeEnabled` and `privacyCookieExpirationPeriod`) introduced in 6.2